Terminal.Gui/Views/ComboBox.cs

```diff
@@ -423,12 +422,12 @@ private bool CancelSelected ()
 
             if (SelectedItem > -1 && _listview.Source?.Count > 0)
             {
-                _search.Text = _text = _listview.Source.ToList () [SelectedItem].ToString ();
+                Text = _listview.Source.ToList () [SelectedItem].ToString ();
```

Comment:
Need a null conditional access on the array element:

```csharp
Text = _listview.Source.ToList () [SelectedItem]?.ToString ();
```

I also don't love that we use `SelectedItem` to mean `SelectedIndex`.